src/var/clearInterval.js

```diff
@@ -0,0 +1,3 @@
+define(function() {
+	return window.clearInterval;
```

Comment:
> It doesn't affect the built file

It does affect it by creating the variable, in some cases it increases the gzipped size a little.

> We've accidentally used the wrong document over and over and over until we did this.

I believe this was `location` that was breaking Node use? This has been taken care of in 76df9e4e389d80bff410a9e5f08b848de1d21a2f by removing the `browser` environment from `.jshintrc` and just putting `window` as a global there.